### ISSUE TITLE
Details on how to combine TexForce with DPOK or ReFL

### ISSUE THREAD

**Issue by Qiulin-W** [chaofengc/TexForce#4]
Hi,

Nice work! After reading the paper, I'm not clear about the details of how to combine TexForce with DPOK, ReFL and Alignprob.  What is the overall training loss should be?  And It seems not straightforward to combine RL learning objective with direct-backpropagation methods.

Thanks!

**Comment by chaofengc** [comment]
Thanks to your interest. Our method finetunes the text encoder while others focus on finetuning u-net.

The combination is as straightforward as the demo codes:
```
load_model_weights(pipe, './lora_weights/sd14_refl/', 'unet+lora')
load_model_weights(pipe, './lora_weights/sd14_texforce/', 'text+lora')
```

**Comment by Qiulin-W** [comment]
Got it! Thanks so much!
